src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -60,6 +60,177 @@ bool AletheProofPostprocessCallback::update(Node res,
     {
       return addAletheStep(AletheRule::ASSUME, res, res, children, {}, *cdp);
     }
+    // See proof_rule.h for documentation on the SCOPE rule. This comment uses
+    // variable names as introduced there. Instead of (not (and F1 ... Fn))
+    // (cl (not (and F1 ... Fn))) is printed and instead of (=> (and F1 ... Fn)
+    // F) the term (cl (=> (and F1 ... Fn))) is printed while the proof node
+    // remains the same.
+    //
+    // Let (not (and F1 ... Fn))^i denote the repetition of (not (and F1 ...
+    // Fn)) for i times.
+    //
+    // T1:
+    //
+    //   P
+    // ----- ANCHOR    ------- ... ------- AND_POS
+    //  VP1             VP2_1  ...  VP2_n
+    // ------------------------------------ RESOLUTION
+    //               VP2a
+    // ------------------------------------ REORDER
+    //  VP2b
+    // ------ DUPLICATED_LITERALS   ------- IMPLIES_NEG1
+    //   VP3                          VP4
+    // ------------------------------------  RESOLUTION    ------- IMPLIES_NEG2
+    //    VP5                                                VP6
+    // ----------------------------------------------------------- RESOLUTION
+    //                               VP7
+    //
+    // VP1: (cl (not F1) ... (not Fn) F)
+    // VP2_i: (cl (not (and F1 ... Fn)) Fi), for i = 1 to n
+    // VP2a: (cl F (not (and F1 ... Fn))^n)
+    // VP2b: (cl (not (and F1 ... Fn))^n F)
+    // VP3: (cl (not (and F1 ... Fn)) F)
+    // VP4: (cl (=> (and F1 ... Fn) F) (and F1 ... Fn)))
+    // VP5: (cl (=> (and F1 ... Fn) F) F)
+    // VP6: (cl (=> (and F1 ... Fn) F) (not F))
+    // VP7: (cl (=> (and F1 ... Fn) F) (=> (and F1 ... Fn) F))
+    //
+    // The reorder step is not necessary if n = 1, because in that case VP2a is
+    // (cl (not F1 F) which is the same as VP2b.
```

Comment:
```suggestion
    // Note that if n = 1, then the ANCHOR step yields (cl (not F1) F), which is
    // the same as VP3. Since VP1 = VP3, the steps for that transformation are
    // not generated.
```